src/main.jl

```diff
@@ -79,8 +81,12 @@ function main(
                 cc_user=cc_user,
                 bump_version=bump_version,
             )
+
+            if !isnothing(pr)
```

Comment:
Do we have a standard for using `isnothing()` vs `=== nothing`?

Reply:
I thought the current standard was to use `isnothing`

Reply:
Someone told me than in 1.7, `isnothing` is defined in terms of `===`, so either should be fine.

Just don't use `==`, I think.